### PR TITLE
Feature/moses/right section home page

### PR DESCRIPTION
**Pull request by SapeleD3**

### CODE REVIEW

File: src/components/feed/components/Feed.jsx [hunk 1]
```diff
@@ -1,10 +1,46 @@
...
+ * function used to render Feed component
+ *
+ * @function {*} Feed
+ */
 const Feed = () => {
     return (
         <div>
             <Navigation />
+            <div className="p-feedPage">
+                {/* <div>{Right}</div> */}
```
Comment (EngrEric) [inline src/components/feed/components/Feed.jsx]: remove unused code

File: src/components/feed/components/right/right.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.number {
+    margin: 0;
+    font-size: 21px;
+    font-weight: 600;
+}
+
+.item2 {
+    flex: 1;
```
Comment (gideonnnalue) [inline src/components/feed/components/right/right.scss]: try to use ABEM standard for naming your classes. And also make use of SCSS class names nesting

File: src/components/feed/components/right/Right.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+ */
+const Right = () => (
+    <div>
+        <div className="p-right">
+            <div className="">{renderImage()}</div>
+            <p className="userName">{userNameText}</p>
+            {renderButton()}
+            {renderGrid()}
+        </div>
+    </div>
```
Comment (enyeInc) [inline src/components/feed/components/right/Right.jsx]: remove unneeded `div`

File: src/components/feed/components/rightDown/RightDown.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * @function {*} Feed
+ */
+const Right = () => (
+    <div>
+        <div className="p-right">
+            <h2>{title}</h2>
+            {topics.map(topic => <p className='topics' key={topic.id}>{topic}</p>)}
+        </div>
+    </div>
```
Comment (enyeInc) [inline src/components/feed/components/rightDown/RightDown.jsx]: remove unneeded `div`

File: src/components/feed/components/right/Right.jsx (hunk 3 above)
Comment (enyeInc) [reply src/components/feed/components/right/Right.jsx]: this was never fixed

**Pull request merged**